huaweicloud/resource_huaweicloud_cbr_policy.go

```diff
@@ -0,0 +1,232 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/cbr/v3/policies"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func resourceCBRPolicyV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCBRPolicyV3Create,
+		Read:   resourceCBRPolicyV3Read,
+		Update: resourceCBRPolicyV3Update,
+		Delete: resourceCBRPolicyV3Delete,
+
```

Comment:
can the CBR policy support to be imported ?

Reply:
It is supported now.